yt/yt/server/lib/exec_node/config.h

```diff
@@ -664,11 +664,42 @@ DEFINE_REFCOUNTED_TYPE(TNbdConfig)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+DEFINE_ENUM(EJobProxyLoggingMode,
+    (Sandbox)
+    (SeparateDirectory)
+);
+
+////////////////////////////////////////////////////////////////////////////////
+
+class TJobProxyLoggingEnvironment
```

Comment:
I think that `Environment` as suffix can be omitted here, but it worth to add`Config` suffix